### ISSUE TITLE
Inconsistent handling of variables in 

### ISSUE THREAD

**Issue by webbpinner** [rvdata/NavManager#2]
I'm digging through the `@navqa(
        $navBestResPreQC, $dateStringUTCStart, $dateStringUTCEnd,
        $speedHoriMax, $accelHoriMax, $gapThreshold, 
        $portLongitudeStart, $portLatitudeStart, 
        $portLongitudeEnd, $portLatitudeEnd,
        $fqalog
    )` function in `navqa.inc.php` and I'm seeing some inconsistencies in how variables are push to the out put.  Below is a var_dump() of the resulting array from the function:

```
object(stdClass)#11 (2) {
  ["duration_and_range_of_values"]=>
  object(stdClass)#1 (20) {
    ["Epoch_Interval"]=>
    object(stdClass)#2 (2) {
      ["uom"]=>
      string(1) "s"
      ["value"]=>
      string(5) "0.500"
    }
    ["Maximum_Altitude"]=>
    object(stdClass)#3 (2) {
      ["uom"]=>
      string(1) "m"
      ["value"]=>
      float(2.48)
    }
    ["Minimum_Altitude"]=>
    object(stdClass)#4 (2) {
      ["uom"]=>
      string(1) "m"
      ["value"]=>
      float(-6.42)
    }
    ["Minimum_Horizontal_Speed"]=>
    object(stdClass)#5 (2) {
      ["uom"]=>
      string(3) "m/s"
      ["value"]=>
      string(4) "0.01"
    }
    ["Maximum_Horizontal_Speed"]=>
    object(stdClass)#6 (2) {
      ["uom"]=>
      string(3) "m/s"
      ["value"]=>
      string(4) "6.06"
    }
    ["Minimum_Horizontal_Acceleration"]=>
    object(stdClass)#7 (2) {
      ["uom"]=>
      string(5) "m/s^2"
      ["value"]=>
      string(6) "-0.611"
    }
    ["Maximum_Horizontal_Acceleration"]=>
    object(stdClass)#8 (2) {
      ["uom"]=>
      string(5) "m/s^2"
      ["value"]=>
      string(5) "0.559"
    }
    ["distanceFromPortStart"]=>
    int(0)
    ["distanceFromPortEnd"]=>
    int(0)
    ["First_Epoch"]=>
    string(24) "2012-04-28T00:00:00.487Z"
    ["Last_Epoch"]=>
    string(24) "2012-04-29T15:03:05.089Z"
    ["Possible_Number_of_Epochs_with_Observations"]=>
    float(281170)
    ["Actual_Number_of_Epochs_with_Observations"]=>
    int(274776)
    ["Actual_Countable_Number_of_Epoch_with_Observations"]=>
    int(274777)
    ["Absent_Number_of_Epochs_with_Observations"]=>
    float(6394)
    ["Flagged_Number_of_Epochs_with_Observations"]=>
    int(0)
    ["Maximum_Number_of_Satellites"]=>
    int(11)
    ["Minimum_Number_of_Satellites"]=>
    int(5)
    ["Maximum_HDOP"]=>
    float(2.2)
    ["Minimum_HDOP"]=>
    float(0.7)
  }
  ["quality_assessment"]=>
  object(stdClass)#9 (7) {
    ["longest_epoch_gap"]=>
    object(stdClass)#10 (2) {
      ["uom"]=>
      string(1) "s"
      ["value"]=>
      string(5) "3.500"
    }
    ["number_of_gaps_longer_than_threshold"]=>
    int(0)
    ["number_of_epochs_out_of_sequence"]=>
    int(0)
    ["number_of_epochs_with_bad_gps_quality_indicator"]=>
    NULL
    ["number_of_horizontal_speeds_exceeding_threshold"]=>
    int(0)
    ["number_of_horizontal_accelerations_exceeding_threshold"]=>
    int(0)
    ["percent_completeness"]=>
    string(5) "97.73"
  }
}
```

I really like the breakout of ['uom'] and ['value'] however it's not used everywhere (see ["distanceFromPortStart"] ).  For consistency sake it would be nice to include the two array elements for all measurements, even those whose uom is non-physical (i.e. ["Maximum_Number_of_Satellites"])

For variables where there is no real uom, setting the value to `null` would work nicely.

With regards to ['value'], some are set to their native value (int, float), others are converted to strings.  I'd be in favor of leaving the values in their native format.  This would simplify any latter mathematical operations.

This example also shows another inconsistency distanceFromPortStart (camelCase) vs Maximum_Number_of_Satellites ("_" demilinated).  number_of_horizontal_speeds_exceeding_threshold is a third format style (lower case) which again it would be nice to homogenize the variable naming scheme.

So to recap...
- it would be great to get a value and uom for each output variable.
- it would be great to leave the variable value in it's native format
- it would be great to standardize the variable naming scheme, my preferred format is camelCase.

Thanks,
Webb
